app/src/main/java/nerd/tuxmobil/fahrplan/congress/contract/BundleKeys.java

```diff
@@ -39,24 +39,10 @@
             "nerd.tuxmobil.fahrplan.congress.ChangesDialog.VERSION";
 
     // Shared Preferences
```

Comment:
Looks like this comment should be removed now.

Reply:
Thanks. Renamed in 19389b35329244512c2d79c362a6d1eb9dea48c5.